console/src/main/webapp/manager/app/services/delegations.es6

```diff
@@ -0,0 +1,24 @@
+angular.module('manager')
+  .factory('Delegations', ['$resource', 'CONSOLE_PRIVATE_PATH', ($resource, baseUri) =>
+    $resource(baseUri + 'delegation/delegations', {}, {
+      query: {
+        cache: true,
+        method: 'GET',
+        // isArray: true
```

Comment:
I think you can remove this comment ?

Reply:
Yep, this will replace the following line when @Vampouille will address [this comment](https://github.com/georchestra/georchestra/pull/1969#issuecomment-376935677). :sunglasses: :sunglasses: